e2e/env/.env.example

```diff
@@ -0,0 +1,10 @@
+# Select environment (see @inrupt/internal-test-env for a list of available environments)
+E2E_TEST_ENVIRONMENT="Inrupt Production"
+E2E_TEST_IDP=https://login.inrupt.com
+
+# The following environment variables should be kept in a .env.test.local file
+# Obtain credentials via https://login.inrupt.com/registration.html
+E2E_TEST_CLIENT_ID="<client-id>"
+E2E_TEST_CLIENT_SECRET="<client-secret>"
+E2E_TEST_USER="<username>"
+E2E_TEST_PASSWORD="<password>"
```

Comment:
Are these actually required in these tests?

Reply:
Good point, not yet because I didn't add any browser-based tests.

Reply:
Fixed with https://github.com/inrupt/solid-client-errors-js/pull/188/commits/01af408b13ef8ce7f2a742721875e0720bad75df